packages/jokul/src/core/tokens/color/system.color.tokens.json

```
@@ -3,14 +3,187 @@
         "background": {
             "page": {
                 "light": {
-                    "value": "{color.brand.sand}",
+                    "value": "{color.brand.96}",
```

Hvis disse skal peke på de nye variablene må vi ha med `ref` som første ledd i navnet på tokenet, ettersom det er med i JSON-strukturen.

```suggestion
                    "value": "{ref.color.brand.96}",
```